src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Components/ComponentDocumentClassifierPass.cs

```diff
@@ -123,10 +123,12 @@ protected override CodeTarget CreateTarget(RazorCodeDocument codeDocument, Razor
                     continue;
                 }
 
+                var constraints = typeParamNode.Tokens.Skip(1).FirstOrDefault();
```

Comment:
What is this parameter we're skipping? Please leave a comment.

Side note: at some point we need to remove all of these IEnumerables and make them ImmutableArrays/Spans for this type of thing.